index.js

```diff
@@ -0,0 +1,306 @@
+(function(f) {
+
+  'use strict';
+
+  /* istanbul ignore else */
+  if (typeof module === 'object' && typeof module.exports === 'object') {
+    module.exports = f();
+  } else if (typeof define === 'function' && define.amd != null) {
+    define([], f);
+  } else {
+    self.sanctuaryTypeClasses = f();
+  }
+
+}(function() {
+
+  'use strict';
+
+  var STC = {};
+  var instances = STC.instances = {Array: {}, Function: {}};
+
+  var FL = {
+    ap: 'ap',
+    chain: 'chain',
+    concat: 'concat',
+    empty: 'empty',
+    equals: 'equals',
+    extend: 'extend',
+    extract: 'extract',
+    map: 'map',
+    of: 'of',
+    reduce: 'reduce',
+    sequence: 'sequence'
+  };
```

Comment:
Why not depend on [`fantasy-land`](https://github.com/fantasyland/fantasy-land/blob/master/index.js) to get these constants? A change to them would probably be released as a major version, and you can always do like I did; [provide *both* the "easy" names, and the fantasy-land prefixed names](https://github.com/Avaq/Fluture/blob/master/fluture.js#L498-L521).